src/test-utils/image-utils/diff-images.js

```diff
@@ -1,60 +1,68 @@
-/*
- * Calculate perceived color difference using YIQ NTSC transmission color space
- * Based on 2010 paper by Yuriy Kotsarenko and Fernando Ramos
- * http://www.progmat.uaem.mx:8080/artVol2Num2/Articulo3Vol2Num2.pdf
- */
-const DEFAULT_THRESHOLD = 255 * 0.05;
-
-const getY = (r, g, b) => r * 0.29889531 + g * 0.58662247 + b * 0.11448223;
-const getI = (r, g, b) => r * 0.59597799 - g * 0.2741761 - b * 0.32180189;
-const getQ = (r, g, b) => r * 0.21147017 - g * 0.52261711 + b * 0.31114694;
-const getESq = (dY, dI, dQ) => 0.5053 * dY * dY + 0.299 * dI * dI + 0.1957 * dQ * dQ;
-
-// Get blended r/g/b value after applying alpha
-const applyAlpha = (c, a) => 255 + (c - 255) * a / 255;
-
-/**
- * Get dE square at given index from two pixel arrays
- * @param {Uint8ClampedArray} img1 - pixel data of first image
- * @param {Uint8ClampedArray} img2 - pixel data of second image
- * @param {Number} i - pixel index
- */
-function colorDelta(img1, img2, index) {
-  return Math.sqrt(colorDeltaSq(img1, img2, index));
+/* global Buffer */
+import fs from 'fs';
+import {PNG} from 'pngjs';
```

Comment:
Can potentially be replaced with `@loaders.gl/images`?

Reply:
Yes, good opportunity to get some initial test use of that module. 

FWIW, `@loaders.gl/images` does have as main requirement that loaded images can be used as input data to headless gl textures, which is not critical here but hopefully this will not be an issue.